static/js/place/chart.tsx

```diff
@@ -523,6 +589,35 @@ class Chart extends React.Component<ChartPropType, ChartStateType> {
           );
         }
         break;
+      case chartTypeEnum.RANKING:
+        if (this.props.rankingChartData) {
+          this.props.rankingChartData
+            .then((rankingChartData) => {
+              if (_.isEmpty(this.props.statsVars)) {
+                this.setState({ display: false });
+                return;
+              }
+              const sv = this.props.statsVars[0];
+              const svData = rankingChartData[sv];
+              for (const data of svData.data) {
```

Comment:
should check that svData is not null or undefined before calling `svData.data`.
Maybe can move lines 606-610 before this for loop and just `return` inside the `if` statement after setting display to false.

Reply:
Done